### PR TITLE
[FLINK-19749][docs] Improve documentation in 'Table API' page

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

*Improve documentation in 'Table API' page*

## Brief change log

*Improve documentation in 'Table API' page*
  - *MD file location: docs/dev/table/tableAPI.md*
  - *page URL: https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/table/tableApi.html*

## Verifying this change

*Improve documentation in 'Table API' page*

This change added tests and can be verified as follows:

  - *Improve documentation in 'Table API' page*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by RocMarshal** [comment]
ping @dianfu

**Review by dianfu — CHANGES_REQUESTED** [review]
@RocMarshal Thanks a lot for the PR. LGTM overall. Could you also update the corresponding Chinese documentation(tableApi.zh.md)?

**Comment by RocMarshal** [comment]
@dianfu 
Done.

File: docs/dev/table/tableApi.zh.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ under the License.
...
 
-The Table API shares many concepts and parts of its API with Flink's SQL integration. Have a look at the [Common Concepts & API]({{ site.baseurl }}/dev/table/common.html) to learn how to register tables or to create a `Table` object. The [Streaming Concepts](./streaming) pages discuss streaming specific concepts such as dynamic tables and time attributes.
+The Table API shares many concepts and parts of its API with Flink's SQL integration. Have a look at the [Common Concepts & API]({% link dev/table/common.md %}) to learn how to register tables or to create a `Table` object. The [Streaming Concepts](./streaming) pages discuss streaming specific concepts such as dynamic tables and time attributes.
```
Comment (dianfu) [inline docs/dev/table/tableApi.zh.md:27]:
```suggestion
The Table API shares many concepts and parts of its API with Flink's SQL integration. Have a look at the [Common Concepts & API]({% link dev/table/common.zh.md %}) to learn how to register tables or to create a `Table` object. The [Streaming Concepts](./streaming) pages discuss streaming specific concepts such as dynamic tables and time attributes.
```
Same for the other places.

**Comment by RocMarshal** [comment]
ping @dianfu @wuchong

File: docs/dev/table/tableApi.zh.md [hunk 2]
```diff
@@ -477,11 +477,11 @@ val result = orders.where($"b" === "red")
   <tbody>
   	<tr>
   		<td>
-        <strong>From</strong><br>
-        <span class="label label-primary">批处理</span> <span class="label label-primary">流处理</span>
+        <strong>FromPath</strong><br>
+        <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span>
```
Comment (dianfu) [inline docs/dev/table/tableApi.zh.md:481]: why change this?

Comment (RocMarshal) [inline docs/dev/table/tableApi.zh.md:481]:
```suggestion
        <span class="label label-primary">批处理</span> <span class="label label-primary">流处理</span>
```

Comment (RocMarshal) [reply docs/dev/table/tableApi.zh.md]: @dianfu So sorry for this issue.
Thank you for your reminder.

**Review by dianfu — APPROVED** [review]
@RocMarshal Thanks a lot for the update. LGTM.

**Pull request closed without merging**